src/OpenTelemetry.Resources.Process/ProcessDetector.cs

```diff
@@ -14,11 +14,22 @@ internal sealed class ProcessDetector : IResourceDetector
     /// <returns>Resource with key-value pairs of resource attributes.</returns>
     public Resource Detect()
     {
+        var currentProcess = System.Diagnostics.Process.GetCurrentProcess();
         return new Resource(new List<KeyValuePair<string, object>>(2)
         {
             new(ProcessSemanticConventions.AttributeProcessOwner, Environment.UserName),
+            new(ProcessSemanticConventions.AttributeProcessCommandLine, Environment.CommandLine),
+            new(ProcessSemanticConventions.AttributeProcessCommandArgs, Environment.GetCommandLineArgs()),
+            new(ProcessSemanticConventions.AttributeProcessArgsCount, Environment.GetCommandLineArgs().Length),
```

Comment:
Should these be opt-in/opt-out in case the arguments passed to the process include any secrets?

Reply:
From [the documentation](https://opentelemetry.io/docs/specs/semconv/registry/attributes/process/):

> SHOULD NOT be collected by default unless there is sanitization that excludes sensitive data.

Reply:
I am fully supporting these comments. Security is crucial here.